src/schema/resolvers.js

```diff
@@ -593,16 +593,31 @@ module.exports = {
         throw err;
       }
 
+      // Fetch oracle to get the finalized result
+      const oracle = await Oracles.findOne({ address: oracleAddress }, { options: 1, optionIdxs: 1 });
+      let winningIndex;
+      if (!oracle) {
+        logger.error(`Could not find Oracle ${address} in DB.`);
```

Comment:
we should ensure oracle exists before call smart contract. If oracle not found, throw an error to FE

Reply:
done